client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -159,6 +160,8 @@ function MetadataListEditingDirective() {
         templateUrl: 'scripts/superdesk-authoring/metadata/views/metadata-terms.html',
         link: function(scope) {
 
+            subjectScope = scope;
```

Comment:
Sorry, my bad, I overlooked it up there. Didn't expect it between the function declaration and its docstring.